packages/elements/stencil.config.ts

```diff
@@ -4,6 +4,7 @@ import { sass } from '@stencil/sass';
 export const config: Config = {
   buildEs5: false, // see https://github.com/ionic-team/stencil/issues/253
   globalStyle: './src/components/styles/variables.scss',
+  globalScript: './src/util/import-fonts.ts',
```

Comment:
There still seems to be a problem with angular not executing the script `globalScript` on load. It's working correctly for the react and storybook package.

Reply:
I will wait for #85 and try again with the newest version.

Reply:
Found another solution